_layouts/download.html

```diff
@@ -20,41 +20,36 @@ <h1>{{page.title}}</h1>
   <div class="wrap">
```

Comment:
I think most of the content on this page is redundant with the index page (the main two links point to getting-started.html and download/all.html, which are already provided on the index page. The other links are also provided on the index page, and they are not really related with the action of “downloading” Scala). I wonder if we should just remove this page? (To do it properly, the URL https://scala-lang.org/download/ should still work, but it should redirect to https://docs.scala-lang.org/getting-started.html)

Reply:
Resolved after internal discussion: let’s keep it for now and see if we can go further in a separate PR.